xbmc/addons/interfaces/GUI/General.cpp

```diff
@@ -0,0 +1,730 @@
+
+    case ACTION_SUBTITLE_VSHIFT_UP     : return ADDON_ACTION_SUBTITLE_VSHIFT_UP;
+    case ACTION_SUBTITLE_VSHIFT_DOWN   : return ADDON_ACTION_SUBTITLE_VSHIFT_DOWN;
+    case ACTION_SUBTITLE_ALIGN         : return ADDON_ACTION_SUBTITLE_ALIGN;
+
+    case ACTION_FILTER                 : return ADDON_ACTION_FILTER;
+
+    case ACTION_SWITCH_PLAYER          : return ADDON_ACTION_SWITCH_PLAYER;
+
+    case ACTION_STEREOMODE_NEXT        : return ADDON_ACTION_STEREOMODE_NEXT;
+    case ACTION_STEREOMODE_PREVIOUS    : return ADDON_ACTION_STEREOMODE_PREVIOUS;
+    case ACTION_STEREOMODE_TOGGLE      : return ADDON_ACTION_STEREOMODE_TOGGLE;
+    case ACTION_STEREOMODE_SELECT      : return ADDON_ACTION_STEREOMODE_SELECT;
+    case ACTION_STEREOMODE_TOMONO      : return ADDON_ACTION_STEREOMODE_TOMONO;
+    case ACTION_STEREOMODE_SET         : return ADDON_ACTION_STEREOMODE_SET;
+
+    case ACTION_SETTINGS_RESET         : return ADDON_ACTION_SETTINGS_RESET;
+    case ACTION_SETTINGS_LEVEL_CHANGE  : return ADDON_ACTION_SETTINGS_LEVEL_CHANGE;
+
+    case ACTION_TRIGGER_OSD            : return ADDON_ACTION_TRIGGER_OSD;
+    case ACTION_INPUT_TEXT             : return ADDON_ACTION_INPUT_TEXT;
+    case ACTION_VOLUME_SET             : return ADDON_ACTION_VOLUME_SET;
+    case ACTION_TOGGLE_COMMSKIP        : return ADDON_ACTION_TOGGLE_COMMSKIP;
+
+    case ACTION_TOUCH_TAP              : return ADDON_ACTION_TOUCH_TAP;
+    case ACTION_TOUCH_TAP_TEN          : return ADDON_ACTION_TOUCH_TAP_TEN;
+    case ACTION_TOUCH_LONGPRESS        : return ADDON_ACTION_TOUCH_LONGPRESS;
+    case ACTION_TOUCH_LONGPRESS_TEN    : return ADDON_ACTION_TOUCH_LONGPRESS_TEN;
+
+    case ACTION_GESTURE_NOTIFY         : return ADDON_ACTION_GESTURE_NOTIFY;
+    case ACTION_GESTURE_BEGIN          : return ADDON_ACTION_GESTURE_BEGIN;
+    case ACTION_GESTURE_ZOOM           : return ADDON_ACTION_GESTURE_ZOOM;
+    case ACTION_GESTURE_ROTATE         : return ADDON_ACTION_GESTURE_ROTATE;
+    case ACTION_GESTURE_PAN            : return ADDON_ACTION_GESTURE_PAN;
+
+    case ACTION_GESTURE_SWIPE_LEFT     : return ADDON_ACTION_GESTURE_SWIPE_LEFT;
+    case ACTION_GESTURE_SWIPE_LEFT_TEN : return ADDON_ACTION_GESTURE_SWIPE_LEFT_TEN;
+    case ACTION_GESTURE_SWIPE_RIGHT    : return ADDON_ACTION_GESTURE_SWIPE_RIGHT;
+    case ACTION_GESTURE_SWIPE_RIGHT_TEN: return ADDON_ACTION_GESTURE_SWIPE_RIGHT_TEN;
+    case ACTION_GESTURE_SWIPE_UP       : return ADDON_ACTION_GESTURE_SWIPE_UP;
+    case ACTION_GESTURE_SWIPE_UP_TEN   : return ADDON_ACTION_GESTURE_SWIPE_UP_TEN;
+    case ACTION_GESTURE_SWIPE_DOWN     : return ADDON_ACTION_GESTURE_SWIPE_DOWN;
+    case ACTION_GESTURE_SWIPE_DOWN_TEN : return ADDON_ACTION_GESTURE_SWIPE_DOWN_TEN;
+
+    case ACTION_ANALOG_MOVE_X          : return ADDON_ACTION_ANALOG_MOVE_X;
+    case ACTION_ANALOG_MOVE_Y          : return ADDON_ACTION_ANALOG_MOVE_Y;
+
+    case ACTION_ERROR                  : return ADDON_ACTION_ERROR;
```

Comment:
Could you please reformat the code to have the case on one line and thne return on the next line - here and in the rest of this source file? I think this is part of Kodi coding guidelines.